archey/entries/model.py

```diff
@@ -179,3 +180,20 @@ def _fetch_android_device_model() -> Optional[str]:
             return None
 
         return f'{brand} ({model})'
+
+    @staticmethod
+    def _fetch_freebsd_model() -> Optional[str]:
+        """Retrieve `vendor` and `product` properties on FreeBSD """
+        try:
+            vendor = check_output(
+                ['kenv', 'smbios.bios.vendor'],
+                universal_newlines=True
+            ).rstrip()
+            product = check_output(
+                ['kenv', 'smbios.system.product'],
+                universal_newlines=True
+            ).rstrip()
+        except FileNotFoundError:
```

Comment:
Shouldn't we catch `CalledProcessError` too, just in case `kenv` fails and/or corresponds to another program ?

Reply:
Yes, [subprocess.checkoutput()](https://docs.python.org/3/library/subprocess.html?highlight=subprocess%20check_output#subprocess.check_output) can indeed raise `CalledProcessError`. I'll add it 👍